### ISSUE TITLE
tslint isn't called

### ISSUE THREAD

**Issue by comods** [Gruntfuggly/formatOnIdle#2]
Please also run the tslint formater when you trigger formats.

Maybe you could enable a config setting where users can add any extention command they need, allowing them to run the format fixer of any rare extention.

**Comment by Gruntfuggly** [comment]
Currently it just uses the `vscode.executeFormatDocumentProvider` to format the document.
How is the tslint formatter normally run? 

I'll add a configuration setting to allow a custom formatter to be run insetad and use the above as a default...

**Comment by comods** [comment]
Just so there isn't a confustion.

1) I still need the normal formaters to run. But thinking about it, maybe others don't. A setting to toggle if vscode.executeFormatDocumentProvider is run would be useful.

2) TSLint is just 1 formater I have run into but like above, I want multiple to run. Could your new setting allow a list, instead of just a single custom formatter?

Thanks for your quick response!

**Comment by Gruntfuggly** [comment]
Yes, both those points make sense - it's probably worth adding two lists - commands to be run before the document formatter, and comands to be run after...
